Core/include/Acts/Vertexing/AdaptiveMultiVertexFitter.ipp

```diff
@@ -321,8 +321,15 @@ bool Acts::AdaptiveMultiVertexFitter<
   for (auto vtx : state.vertexCollection) {
     Vector3 diff = state.vtxInfoMap[vtx].oldPosition.template head<3>() -
                    vtx->fullPosition().template head<3>();
-    SymMatrix3 vtxWgt =
-        (vtx->fullCovariance().template block<3, 3>(0, 0)).inverse();
+    ActsMatrix<3, 3> tmp_inverse = ActsMatrix<3, 3>::Zero();
+    bool is_invertible_vtx_cov = false;
+    SymMatrix3 vtxWgt = SymMatrix3::Zero();
```

Comment:
what happens if we continue with a zero covariance?

also shouldn't we log an error here?